tests/unit/modules/test_mysql.py

```diff
@@ -620,6 +620,29 @@ def test_plugin_status(self):
             "auth_socket",
         )
 
+    def test_sanitize_comment(self):
+        """
+        Test comment sanitization
+        """
+        input_data = """/*
+        multiline
+        comment
+        */
+        CREATE TABLE test_update (a INT); # end of line comment
+        # example comment
+        insert into test_update values (1); -- ending comment
+        -- another comment type
```

Comment:
Just one more case please-please. A comment sign inside the quotes.

Reply:
Is this what you meant? (see commit)

Reply:
Perfect! Thank you!